src/java.desktop/macosx/native/libawt_lwawt/java2d/metal/MTLGraphicsConfig.m

```diff
@@ -149,20 +149,6 @@ + (void) _getMTLConfigInfo: (NSMutableArray *)argValue {
 
     NSAutoreleasePool* pool = [[NSAutoreleasePool alloc] init];
 
-
```

Comment:
Please also check how this function is called, looks like previously it was called as a selector+an array as a parameter, and then reworked as a performOnMainThreadWaiting+block, but it still use an array as a parameter. I think it is similar to JDK-8238075.

Reply:
Excellent point! Thanks for the pointer to the bug. 
A lot of code in this file can be cleaned up. I will update the PR soon.